tasks/theo-agent.yml

```diff
@@ -1,10 +1,19 @@
 ---
+- name: Check variables consistency
+  assert:
+    that: ((theo_verify_signature|bool and theo_public_key|string|length|int > 0 and theo_public_key|string != 'None') or not theo_verify_signature|bool)
```

Comment:
Out of curiosity.... Is there a reason for testing `theo_public_key|string != 'None'` and not any other invalid public key string which length is `>0`?

Reply:
I didn't want to introduce now a formal check on theo_public_key value. This is just a check to see, when theo_verify_signature is True, if a public key has been set.
The problem I faced is when theo_public_key is not defined at all. Casting a null var to string ends in a string of value 'None'.
If you have any better solution to replace this logic, it's welcome!

Reply:
Fair enough!